docs/es/docs/tutorial/path-params.md

```diff
@@ -0,0 +1,244 @@
+# Parámetros de path
+
+Puedes declarar los "parámetros" o "variables" con la misma sintaxis que usan los format strings de Python:
```

Comment:
Do we use underscore on foreign languages? If so:

```suggestion
Puedes declarar los "parámetros" o "variables" con la misma sintaxis que usan los *format* strings de Python:
```

Reply:
Yeah, it makes sense, but at the same time, there's already a custom format for FastAPI-specific terms, like *path operation*, and for `variables` and `code`, so for now I'll leave it as it is.